service/server.py

```diff
@@ -137,6 +131,23 @@ def get_title(title_number):
         abort(404)
 
 
+@app.route('/titles/<title_number>.pdf', methods=['GET'])
+@login_required
+def display_title_pdf(title_number):
+    if _show_full_title_pdf():
+        full_title_data = api_client.get_official_copy_data(title_number)
+        if full_title_data:
+            sub_registers = full_title_data.get('official_copy_data', {}).get('sub_registers')
+            if sub_registers:
+                publication_date = datetime.datetime(3001, 2, 3, 4, 5, 6)  # TODO: get real date
```

Comment:
I think we need something so the designer can see it (as well as other people in the organisation). It has an obviously wrong value and a TODO comment so hopefully it's not too bad.